classes/notification/types/MissionNotificationDto.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+class MissionNotificationDto extends NotificationDto {
+    public string $mission_rank;
+
+    public static function fromDb($row, $action_url) {
+        $attributes = json_decode($row['attributes'], true);
+        $notification = new MissionNotificationDto();
+        $notification->type = $row['type'];
+        $notification->message = $row['message'];
+        $notification->notification_id = $row['notification_id'];
+        $notification->user_id = $row['user_id'];
+        $notification->created = $row['created'];
+        $notification->duration = $row['duration'];
+        $notification->alert = $row['alert'];
+        $notification->action_url = $action_url;
+        $notification->mission_rank = $attributes['mission_rank'];
```

Comment:
Do we need mission ID too?

Reply:
Just `mission_rank` for the display, other necessary info (mission name, location) passed to message.